### DISCUSSION TITLE
Please add an option to disable LongPress functions when they are not needed (LongLeft, LongRun, LongDown, LongUp, LongSelect)

### DISCUSSION THREAD

**Discussion by sasha-ld** [react-native-tvos/react-native-tvos#676] [Feature Requests]
### Description

Please add an option to disable LongPress functions when they are not needed (`longLeft`, `longRight`, `longDown`, `longUp`, `longSelect`).
I want to implement a custom progress bar and video navigation, but when I long-press the remote control button, it triggers the `longLeft` or `longRight` events, which occur every 300ms, which is quite slow for fast video navigation. It would be possible to handle this event if there were events like `rightUp` or `leftUp`, but such events do not exist.
I Am use Android TV

### React Native Version

0.73.2-0

### Output of `npx react-native info`

```
info Fetching system and libraries information...
System:
  OS: Windows 11 10.0.22631
  CPU: (8) x64 Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
  Memory: 4.39 GB / 15.96 GB
Binaries:
  Node:
    version: 20.10.0
    path: C:\Program Files\nodejs\node.EXE
  Yarn:
    version: 1.22.21
    path: ~\AppData\Roaming\npm\yarn.CMD
  npm:
    version: 10.2.3
    path: C:\Program Files\nodejs\npm.CMD
  Watchman: Not Found
SDKs:
  Android SDK:
    API Levels:
      - "31"
      - "33"
      - "34"
    Build Tools:
      - 33.0.0
      - 33.0.1
      - 34.0.0
    System Images:
      - android-28 | Android TV Intel x86 Atom
      - android-30 | Android TV Intel x86 Atom
      - android-31 | Android TV Intel x86 Atom
      - android-33 | Android TV Intel x86 Atom
      - android-33 | Intel x86_64 Atom
      - android-33 | Google APIs Intel x86_64 Atom
      - android-33 | Google Play Intel x86_64 Atom
      - android-34 | Android TV Intel x86 Atom
      - android-34 | Google APIs Intel x86_64 Atom
    Android NDK: Not Found
  Windows SDK: Not Found
IDEs:
  Android Studio: AI-231.9392.1.2311.11330709
  Visual Studio: Not Found
Languages:
  Java:
    version: 17.0.8
    path: C:\Program Files\Microsoft\jdk-17.0.8.7-hotspot\bin\javac.EXE
  Ruby: Not Found
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native: Not Found
  react-native-tvos:
    installed: 0.73.2-0
  react-native-windows: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: false
iOS:
  hermesEnabled: Not found
  newArchEnabled: Not found
```

### Steps to reproduce

```
const myTVEventHandler = evt => {
    if (focus == 'progress'){
      if (evt.eventType == 'right'){
        console.log(evt.eventType)
        !pause ? setPause(true) : null
        setCurrentTime(currentTime+30)
      }else if( evt.eventType == 'left'){
        console.log(evt.eventType)
        !pause ? setPause(true) : null
        setCurrentTime(currentTime-30)
      }else if (evt.eventType == 'select'){
        video.current.seek(currentTime)
        setPause(!pause)
      }else if (evt.eventType == 'longLeft'){
        console.log(evt.eventType)
        !pause ? setPause(true) : null
        setCurrentTime(currentTime-30)
      }else if (evt.eventType == 'longRight'){
        console.log(evt.eventType)
        !pause ? setPause(true) : null
        setCurrentTime(currentTime+30)
      }
    }
     }
  
    useTVEventHandler(myTVEventHandler);
```

### Snack, screenshot, or link to a repository

right events 

https://github.com/react-native-tvos/react-native-tvos/assets/79357766/ed687e37-f974-4f29-892a-0db53e37080d

longRight events

https://github.com/react-native-tvos/react-native-tvos/assets/79357766/9714fe52-3dbf-42c9-83cd-2013b72c7a02

**Comment by douglowder** [comment]
Converted this to a feature request... we will take a look

**Comment by yusufyildirim** [comment]
Technically, I think it's somewhat achievable on Android  with: https://github.com/react-native-tvos/react-native-tvos/blob/43e88d57611a42a60578201f041b710c1e77376e/packages/react-native/ReactAndroid/src/main/java/com/facebook/react/config/ReactFeatureFlags.java#L95

But I do think we should improve our Remote handling module to make it resilient for more use-cases like this one

**Comment by sasha-ld** [comment]
How to apply changes? I launch the project and nothing changes.

**Reply by douglowder** [reply]
You would need to set the above feature flag by making changes to `android/app/src/main/java/com/douglowderexpo/TVInputDemo/MainApplication.kt`, adding the import statement `import com.facebook.react.config.ReactFeatureFlags` and setting `ReactFeatureFlags.enableKeyDownEvents` to true.
